### ISSUE TITLE
Feature Request - Allow to surround all widgets in group with a prefix and surfix?

### ISSUE THREAD

**Issue by marktopper** [Arrilot/laravel-widgets#68]
I would like to surround all widgets in a group with ex. a div tag.

Maybe something like.
```
Widget::group('some-group')->surround(function ($content, $index) {
    return "<div class="widget widget-{$index}">{$content}</div>";
})->display();
```

**Comment by arrilot** [comment]
Looks good to me. I would be happy to accept a PR adding that.

**Comment by marktopper** [comment]
Will work on a pull request.

**Comment by arrilot** [comment]
Hi, I've implemented this feature in a fresh 3.7.2 version
```php
Widget::group('sidebar')->wrap(function ($content, $index, $total) {
    // $total is a total number of widgets in a group.
    return "<div class='widget-{$index}'>{$content}</div>";
})->...;
```
